bliss/catalog.py

```diff
@@ -488,6 +488,10 @@ def to_tile_params(
         for ii in range(self.batch_size):
             n_sources = int(self.n_sources[ii].item())
             for idx, coords in enumerate(tile_coords[ii][:n_sources]):
+                if coords[0] >= tile_n_sources.shape[1] or coords[1] >= tile_n_sources.shape[2]:
+                    continue
+                # ignore sources outside of the image (usually caused by data augmentation - shift)
```

Comment:
It seems preferable to crop the sources after the shift rather than doing it here

Reply:
OK sounds good